packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -1,40 +1,131 @@
 import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faQuestion, faTimes } from '@fortawesome/free-solid-svg-icons';
-import cn from 'classnames';
+import { Box } from '@strapi/parts/Box';
+import { Text } from '@strapi/parts/Text';
+import { FocusTrap } from '@strapi/parts/FocusTrap';
 import { useConfigurations } from '../../../hooks';
-import StaticLinks from './StaticLinks';
-import Wrapper from './Wrapper';
+
+const Wrapper = styled.button`
```

Comment:
Could you name it something a bit more explicit? To me a `Wrapper` is not supposed to be a button